pkg/route/controller/ingress/ingress_test.go

```diff
@@ -635,7 +635,7 @@ func TestController_sync(t *testing.T) {
 							Name: "service-1",
 						},
 						Port: &routev1.RoutePort{
-							TargetPort: intstr.FromInt(8080),
+							TargetPort: "http",
```

Comment:
Since the changes you introduce in `pkg/route/controller/ingress/ingress.go ` support using a port number or name, can you update the test cases accordingly?

Reply:
Actually I did not introduce that feature (this was already there) - just used the right port (which in this case has a name). I agree some more meaningful tests would be good - trying to add some.